chat-memories/spring-ai-jdbc-chat-memory/src/main/resources/org/springframework/ai/chat/memory/jdbc/schema-drop-postgresql.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS ai_chat_memory;
```

Comment:
Same comment as above.

Reply:
This file is now removed. See comment above.